packages/@atjson/react/src/components/Slice.ts

```diff
@@ -26,8 +26,24 @@ export function Slice(props: {
     [value]
   );
 
+  let children = useMemo(() => {
```

Comment:
this is the same code as in packages/@atjson/react/src/index.ts, could we unify it under some helper functions?